samples/scaleout/senderside-and-distributor/sample.md

```diff
@@ -0,0 +1,91 @@
+---
+title: Multi-message conversations when scaling out
+summary: How messages belong to the same conversation are routed depending on the scaling out approach used
+component: Core
+reviewed: 2016-11-30
+tags:
+ - Distributor
+ - Conversation
+ - Session
+related:
+ - nservicebus/msmq/sender-side-distribution
+ - nservicebus/msmq/distributor
+ - samples/scaleout/distributor
+ - samples/scaleout/senderside
+---
+
+The sample demonstrates differences in behavior of some of NServiceBus APIs that depend on the scaling out method.
```

Comment:
Well, it enumerates these APIs in `APIs that depend on scaling out approach` section.